signal2html/core.py

```diff
@@ -265,6 +265,10 @@ def process_backup(backup_dir, output_dir):
 
         t = Thread(_id=_id, recipient=recipient)
         thread_dir = os.path.join(output_dir, t.sanename)
+        i = 2
+        while os.path.exists(thread_dir):
+            thread_dir = os.path.join(output_dir, t.sanename + f"_{i}")
```

Comment:
At the moment, when I have multiple phones for the same contact, they will all be in the same directory. I.e. instead of
John_Doe/
  +12345.html
  +67890.html

I would now have John_Doe_1 and John_Doe_2

Do you prefer it this way?

Reply:
Oh good catch! My intention was to catch potential conflicts where there are, for instance, two similar group names where one ends in an emoji and the other doesn't. That would map to the same ``sanename``, and possibly lead to conflicts. However your example is indeed a case where you would want to merge the directories.

I've pushed a new commit with an attempt to address both cases.

Reply:
Thanks, looks good now!

Reply:
Great, thank you!